README.md

```diff
@@ -1,236 +1,93 @@
 
-## Technical Support or Questions
+### Clone the project you forked
 
-If you have questions or need help integrating the product please [contact us](https://www.creative-tim.com/contact-us) instead of opening an issue.
+The URL of the project you cloned would look like 
+(Replace [your_username] with your github username.)
+``` https://github.com/[your_username]/codeJunction-site```
 
-## Licensing
+You can clone it to your local machine by typing and entering the below command in your terminal. 
+(Replace [your_username] with your github username.)
+``` git clone https://github.com/[your_username]/codeJunction-site```
 
-- Copyright 2018 Creative Tim (https://www.creative-tim.com)
-- Creative Tim [license](https://www.creative-tim.com/license)
+![ezgif com-video-to-gif (2)](https://user-images.githubusercontent.com/35809841/78869616-78c85500-7a62-11ea-91bc-7c62a072bfe9.gif)
 
-## Useful Links
+### Build and Run the Project
 
-- [More products](https://www.creative-tim.com/bootstrap-themes) from Creative Tim
+Move to the project folder
 
-- [Tutorials](https://www.youtube.com/channel/UCVyTG4sCw-rOvB9oHkzZD1w)
+```cd codeJunction-site```
 
-- [Freebies](https://www.creative-tim.com/bootstrap-themes/free) from Creative Tim
+![ezgif com-video-to-gif (3)](https://user-images.githubusercontent.com/35809841/78870157-5420ad00-7a63-11ea-9ee4-92ab3ccda1db.gif)
 
-- [Affiliate Program](https://www.creative-tim.com/affiliates/new) (earn money)
+Run the following command to install the angular cli latest version
+``` npm install -g @angular/cli@latest```
 
-##### Social Media
+Then run an npm install
+``` npm install```
+
+After that, do a npm start 
+``` npm start```
+
+Now the project should be up and running in the default port
+```  http://localhost:4200/```
+
+## Reporting Issues
+We use GitHub Issues as the official bug and new feature tracker for the Official codeJunction website. 
+
+These are the steps you must follow when you are going to report a bug or suggest a new feature to the project.
+1. Go to the issues tab.
+``` https://github.com/codeJunctionTeam/codeJunction-site/issues```
+2. Click "New Issue" Button.
+3. Then you will be prompted a menu with options as "Bug report" and "Feature request".
+4. You can select either option based on the bug or feature you are going to report as an issue.
+5. After selecting the relevant option, provide a title for your issue.
+6. Next step is to fill out the template with accurate information about the bug or feature.
+
+Here are some advices for our users that want to report an issue:
+1. Providing us reproducible steps for the issue will shorten the time it takes for it to be fixed.
+2. Some issues may be browser specific, so specifying in what browser you encountered the issue might help.
+3. If it is a new feature, try to explain it in a manner in which the reader would be able to get a fair understanding about it.
+
+## Licensing
+
+- Copyright 2018 Creative Tim (https://www.creative-tim.com)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/codeJunctionTeam/codeJunction-site/pullRequest?prid=5333118)